cpp/include/cugraph/algorithms.hpp

```diff
@@ -303,6 +303,92 @@ void edge_betweenness_centrality(const raft::handle_t& handle,
                                  vertex_t k               = 0,
                                  vertex_t const* vertices = nullptr);
 
+/**
+ * @brief     Compute betweenness centrality for a graph
+ *
+ * Betweenness centrality for a vertex is the sum of the fraction of
+ * all pairs shortest paths that pass through the vertex.
+ *
+ * The current implementation does not support a weighted graph.
+ *
+ * If @p num_vertices and @p vertices are not specified then this function will compute
+ * exact betweenness (compute betweenness using a traversal from all vertices).
+ *
+ * If @p num_vertices or @p vertices are specified then we will compute approximate
+ * betweenness either by random sampling @p num_vertices as the seeds of the traversal, or
+ * by using the provided @p vertices as the seeds of the traversal.
+ *
+ * If both @p and @num_vertices are specified we will throw an error.
+ *
+ * @throws                 cugraph::logic_error when an error occurs.
+ *
+ * @tparam vertex_t Type of vertex identifiers. Needs to be an integral type.
+ * @tparam edge_t Type of edge identifiers. Needs to be an integral type.
+ * @tparam weight_t Type of edge weights. Needs to be a floating point type.
+ * @tparam multi_gpu Flag indicating whether template instantiation should target single-GPU (false)
+ *
+ * @param handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator, and
+ * handles to various CUDA libraries) to run graph algorithms.
+ * @param graph_view Graph view object.
+ * @param num_vertices Optional, if specified, how many vertices to randomly select
+ * @param vertices Optional, if specified this provides the list of vertices to app
```

Comment:
It seems like some texts after `to app` got deleted.

Reply:
Fixed missing text, cleaned up documentation based on using std::variant